test/error-placement.js

```diff
@@ -291,6 +291,42 @@ test( "test existing non-label used as error element", function(assert) {
 	assert.noErrorFor( field );
 });
 
+test( "test aria-describedby with input names contains special characters (:, ., [, ], $)", function() {
```

Comment:
please note in the test description that we are talking about css-selector meta characters contained in fieldnames